internal/configs/configschema/path.go

```diff
@@ -58,3 +58,19 @@ func (o *Object) AttributeByPath(path cty.Path) *Attribute {
 	}
 	return nil
 }
+
+// PathName is trying to get the name of the attribute from cty.Path.
+// Due to the implementation difference between cty.IndexStep and cty.GetAttrStep,
+// this method is working only with a path where the last element is a cty.GetAttrStep.
+// TODO ephemeral - is there a better way to do this?
+func PathName(path cty.Path) string {
```

Comment:
I'm having trouble squaring the description of this function with the implementation?

It looks like it returns the first GetAttrStep in a path sequence?  Is it supposed to have a break?

Reply:
Good catch! Fixed in 73f00b8193b269081d404b96d96f3ad0545d1047